### DISCUSSION TITLE
How to config s3 generic

### DISCUSSION THREAD

**Discussion by John357John** [requarks/wiki#7446] [Help / Questions]
I want to connect minio, but I get these error:
"Inaccessible host: `test.172.19.0.7' at port `undefined'. This service may not be available in the `us-east-1' region."
The picture is my s3 generic config, i guass it can't identify the port.
![微信图片_20241120092404](https://github.com/user-attachments/assets/3cc0b753-87b1-4f20-b96d-4fd76f452fa2)

**Comment by BPowell76** [comment]
I haven't gotten it to work either, but for me I am using the URI supplied by BackBlaze B2. I'm thinking there's a bug because it fails even with new access keys and I can use those keys for `rclone` and `rsync` with no issues.

**Reply by John357John** [reply]
it seems it needs a port but the interface didnt provide a port, it's a bug probably. It seems it couldn't resolve ip + port.
